package.json

```diff
@@ -35,26 +35,23 @@
   },
   "dependencies": {
     "@nextcloud/router": "^2.0.0",
-    "core-js": "^3.6.4",
     "dompurify": "^2.4.1",
     "escape-html": "^1.0.3",
     "node-gettext": "^3.0.0"
   },
   "devDependencies": {
-    "@happy-dom/jest-environment": "^8.1.0",
     "@nextcloud/browserslist-config": "^2.3.0",
     "@nextcloud/typings": "^1.0.0",
     "@types/node-gettext": "^3.0",
     "check-es-compat": "^2.2.0",
     "gettext-parser": "^4.0.3",
-    "jest": "^27.2.1",
+    "jest": "^29.3.1",
+    "jest-environment-jsdom": "^29.3.1",
```

Comment:
What is the reason for switching to jsdom?
(Asking because happydom should be much faster)

Reply:
Seemed to be recommended with ts-jest and inspired myself from the other configs that works :)
But I have absolutely zero preferences, feel free to switch to happydom. As far as I understood jsdom seemed a bit more mature, but I have little expertise on the subject.

The rest of my config  change should still be relevant :wink:

Reply:
No I am fine with it, I was just curious if there is a reason I do not know :)